### PR TITLE
Fix/windows documentation

### PR DESCRIPTION
**Pull request by jborean93**
##### ISSUE TYPE
- Docs Pull Request
##### COMPONENT NAME

intro_windows.rst
##### SUMMARY

This is a pull request is relation to https://github.com/ansible/ansible/issues/17311 to add in some more information regarding NTLM authentication.

### CODE REVIEW

**Comment by dharmabumstead** [comment]
@nitzmahone Hey Mat - I'm OK to merge this (and do some minor touch ups) if you are OK to sign off on it after you tech review it.

File: docsite/rst/intro_windows.rst [hunk 1]
```diff
@@ -160,17 +168,65 @@ Ansible's windows support relies on a few standard variables to indicate the use
 
 .. include:: ../rst_common/ansible_ssh_changes_note.rst
 
-In group_vars/windows.yml, define the following inventory variables::
+Examples
+++++++++
+
+Below are some examples of how to set your inventory based on the auth method chosen. THese should all be defined in the group_vars/windows.yml file.
```
Comment (alikins) [inline docsite/rst/intro_windows.rst]: s/THese/These

File: docsite/rst/intro_windows.rst [hunk 2]
```diff
@@ -242,6 +289,100 @@ Looking at an Ansible checkout, copy the `examples/scripts/upgrade_to_ps3.ps1 <h
 
 .. _what_windows_modules_are_available:
 
+Verifying Windows Configuration
+```````````````````````````````
+
+After you have set up the Windows host you can use the following commands to verify the setup to ensure Ansible can communite with the host.
```
Comment (alikins) [inline docsite/rst/intro_windows.rst]: communite -> communicate

File: docsite/rst/intro_windows.rst [hunk 3]
```diff
@@ -242,6 +289,100 @@ Looking at an Ansible checkout, copy the `examples/scripts/upgrade_to_ps3.ps1 <h
...
+
+Verify WinRM Listeners
+++++++++++++++++++++++
+
+To see what listeners are currently active you can run ``winrm enumerate winrm/config/listener`` the following command in PowerShell::
```
Comment (alikins) [inline docsite/rst/intro_windows.rst]: I think:

> you can run `winrm enumerate winrm/config/listener` the following command in PowerShell::

should be:

> you can run the following `winrm` command in PowerShell::

File: docsite/rst/intro_windows.rst [hunk 4]
```diff
@@ -242,6 +289,100 @@ Looking at an Ansible checkout, copy the `examples/scripts/upgrade_to_ps3.ps1 <h
...
+       URLPrefix = wsman
+       CertificateThumbprint = 1234567890ABCDEF111213141516171819101A1B
+       ListeningOn = 127.0.0.1, ::1
+
+In the above example we can see both a HTTP and HTTPS listener has been created and are listening on ports 5985 and 5986 respectively.
+
+Verify Service Settings
++++++++++++++++++++++++
+
+Once you have verified that WinRM is setup and listener on a port you can use ``winrm get winrm/config/service`` to get the configuration of the WinRM service::
```
Comment (alikins) [inline docsite/rst/intro_windows.rst]: listener -> listening

File: docsite/rst/intro_windows.rst [hunk 5]
```diff
@@ -242,6 +289,100 @@ Looking at an Ansible checkout, copy the `examples/scripts/upgrade_to_ps3.ps1 <h
...
+           HTTP = 5985
+           HTTPS = 5986
+       IPv4Filter = *
+       IPv6Filter = *
+       EnableCompatibilityHttpListener = false
+       EnableCompatibilityHttpsListener = false
+       CertificateThumbprint
+       AllowRemoteAccess = true
+
+In the above example we are able to authenticate with the Windows host using
```
Comment (alikins) [inline docsite/rst/intro_windows.rst]: (missing ':' )

> In the above example we are able to authenticate with the Windows host using

->

> In the above example we are able to authenticate with the Windows host using:

File: docsite/rst/intro_windows.rst [hunk 6]
```diff
@@ -160,17 +168,65 @@ Ansible's windows support relies on a few standard variables to indicate the use
...
+   ansible_connection: winrm
+   ansible_transport: certificate
+   ansible_winrm_cert_pem: path to the client authentication certificate file path in PEM format
+   ansible_winrm_cert_key_pem: path to the client authentication certificate key file path in PEM format
+
+
+Kerberos Auth
+-------------
+
+Inventory setup for Kerberos auth, note: specifying ansible_password does not make any difference here, you need to have obtained a valid Kerberos ticket outside of Ansible for this to work::
```
Comment (alikins) [inline docsite/rst/intro_windows.rst:201]: is the 'note:' here meant to be a 'note::' directive.

**Review by alikins — COMMENTED** [review]
A few typos, but I'll fix those before merge. Looks good to me otherwise.

**Comment by nitzmahone** [comment]
Sorry, there's a complete Windows docs rewrite that's been sitting half finished on my plate for awhile now. Meantime, there are a number of issues in this text that need to be resolved before we merge it- @jborean93 I see you already nuked your branch, if you want to reopen, I can probably just fix the issues inline with merge faster than telling you and having you fix them, but know this would be a short-term change anyway. (sorry 'bout the shuffle)

**Comment by jborean93** [comment]
If this is a short term change and you have already started a rewrite I don't mind waiting, I just saw the issue and added the sections. I can get the branch back up again if you still wish to continue but I will leave it up to you.

**Comment by dagwieers** [comment]
For some reason, these changes are not in the online documentation.

I thought there was something wrong with generating the documentation, but it seems these changes are no longer in Git either. Why is there no mention of NTLM anymore and what can we do about #17311 in this regard ?

cc @gundalow

**Comment by gundalow** [comment]
@dag looks like @nitzmahone reverted the changed.

**Comment by nitzmahone** [comment]
Online docs just haven't been re-genned yet- usually one of the docs folks does that occasionally, so this will be in the next publish, whenever they do it.

**Comment by dharmabumstead** [comment]
I'll publish this afternoon (PST).

**Pull request merged**